platform/mime/core/platform-mime-core-api/src/main/java/ddf/mime/MimeTypeToTransformerMapper.java

```diff
@@ -15,13 +15,14 @@
 
 import java.util.List;
 import javax.activation.MimeType;
+import javax.annotation.Nullable;
 
 /** @author Ashraf Barakat */
 public interface MimeTypeToTransformerMapper {
 
-  public static final String MIME_TYPE_KEY = "mime-type";
```

Comment:
Not sure if this changes compatibility.

Reply:
I don't think it will. I think `public static final` is implied.